config/config.go

```diff
@@ -0,0 +1,79 @@
+package config
+
+import (
+	"os"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/4rchr4y/gdk/must"
+	"github.com/4rchr4y/gdk/regex"
+)
+
+func MustGetInt(key string) int                { return must.Must(strconv.Atoi(os.Getenv(key))) }
+func MustGetUint(key string) uint64            { return must.Must(strconv.ParseUint(os.Getenv(key), 10, 64)) }
+func MustGetFloat64(key string) float64        { return must.Must(strconv.ParseFloat(os.Getenv(key), 64)) }
+func MustGetString(key string) string          { return must.MustBeOk(os.LookupEnv(key)) }
+func MustGetDuration(key string) time.Duration { return must.Must(time.ParseDuration(os.Getenv(key))) }
+func MustGetUrl(key string) string {
+	value := must.MustBeOk[string](os.LookupEnv(key))
+
+	matched, _ := regexp.MatchString(regex.UrlPatternString, value)
```

Comment:
Here I tihink its better to explicitly handle the error and panic if it occurs. This will be a simpler error for cause-and-investigation analysis than just panic that "smt must not ok"